src/store/metadata/stub/content/editions/versions/cpih_2022-01.json

```diff
@@ -11,37 +11,37 @@
             ],
             "@id": "https://staging.idpd.uk/datasets/cpih/editions/2022-01/versions/1",
             "identifier": "1",
-            "issued": "2017-01-01T00:00:00Z",
+            "issued": "2022-01-01T00:00:00Z",
             "title": "Consumer Prices Index including owner occupiers' housing costs (CPIH)",
-            "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a...",
-            "description": "The Consumer Prices Index...",
+            "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH).",
```

Comment:
Changes to summary and description are great, but they should still differ from the title to some extent.